core/storage/boltdb/storage.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package boltdb
+
+import (
+	"path/filepath"
+
+	"github.com/asdine/storm"
+	"github.com/mysteriumnetwork/node/core/storage"
+)
+
+type bolt struct {
+	db *storm.DB
+}
+
+// NewStorage creates a new BoltDB storage for service promises
```

Comment:
storage can be used for generic objects, for example for sessions too.
Functions comments can be replaced like this:
"for service promises" -> "for storing values"

Reply:
For now, it's used only for storing promises, what is the main aim of the PR. 
Let's replace the comment (and probably not comment only) when we will have something different.